tests/schemas/test_faust_schema.py

```diff
@@ -209,13 +208,21 @@ class Car(faust.Record, AvroModel):
         "A Car"
         engine_name: str
 
+    class TripDistance(enum.Enum):
+        CLOSE = "Close"
+        FAR = "Far"
+
+        class Meta:
+            doc = "Distance of the trip"
+
     class UnionSchema(faust.Record, AvroModel):
         "Some Unions"
         first_union: typing.Union[str, int]
         logical_union: typing.Union[datetime.datetime, datetime.date, uuid.uuid4]
         lake_trip: typing.Union[Bus, Car]
         river_trip: typing.Union[Bus, Car] = None
         mountain_trip: typing.Union[Bus, Car] = dataclasses.field(default_factory=lambda: {"engine_name": "honda"})
+        trip_distance: typing.Union[int, TripDistance] = None
```

Comment:
I think this type annotation is incorrect. Since the default value is `None`, the type should be `Optional`.
```suggestion
        trip_distance: typing.Union[int, TripDistance, None] = None
```

Reply:
done!